app/mailers/gias_establishment_import_mailer.rb

```diff
@@ -4,8 +4,19 @@ def import_notification(user, result)
       "316ef413-5e53-48e4-8a78-2aeaa9b98114",
       to: user.email,
       personalisation: {
-        result: result
+        result: format_result(result)
       }
     )
   end
+
+  private def format_result(result)
+    <<~TEXT
+      Total CSV rows: #{result[:result][:total_csv_rows]}
+      New records: #{result[:result][:new_records]}
+      Changed records: #{result[:result][:changed_records]}
+      Changes: #{result[:result][:changes]}
```

Comment:
Changes has the potential to be a lot - I think we should avoid including it here for now, we need to decide what to do with all this info, if anything, but I am not sure it should go in an email.